dap_job_quality/utils/text_cleaning.py

```diff
@@ -84,6 +84,19 @@ def detect_camelcase(text):
 }
 
 
+def split_on_period_space(text):
+    """
+    A spaCy pipeline component that inserts a space after a period if the period is followed by an uppercase letter or digit
+    without any intervening space, as long it is not preceded by a digit or the pound sign.
```

Comment:
this function doesn't look like it relies on a spacy component? I would update the docstring.

Reply:
Good catch, thank you!